SPECS/mstflint/mstflint.spec

```
@@ -1,7 +1,9 @@
+%global kver %(/bin/rpm -q --queryformat '%{RPMTAG_VERSION}-%{RPMTAG_RELEASE}' $(/bin/rpm -q --whatprovides kernel-devel))
+%global install_mod_dir lib/modules/%{kver}/extra/%{name}
```

Same small update as in the "signed" spec:
```suggestion
%global install_mod_dir %{_libdir}/modules/%{kver}/extra/%{name}-kernel
```